src/vs/platform/accessibility/browser/accessibilityService.ts

```diff
@@ -17,7 +17,7 @@ export class AccessibilityService extends Disposable implements IAccessibilitySe
 
 	private _accessibilityModeEnabledContext: IContextKey<boolean>;
 	protected _accessibilitySupport = AccessibilitySupport.Unknown;
-	protected readonly _onDidChangeScreenReaderOptimized = new Emitter<void>();
+	protected readonly _onDidChangeScreenReaderOptimized = new Emitter<boolean>();
```

Comment:
should we just stick with void since we have an accessor for the new value in the service? I'm concerned that we fire this event with different conditions in different places and feel like we are prone to send an out of sync value in the event compared to what the accessor would give.

Reply:
yeah makes sense

Reply:
we will have to expose `_accessibilitySupport` though as that's what gets set